Jellyfin.Api/Controllers/ItemLookupController.cs

```diff
@@ -274,10 +277,8 @@ public async Task<ActionResult<IEnumerable<RemoteSearchResult>>> GetBookRemoteSe
             }
 
             await DownloadImage(providerName, imageUrl, urlHash, pointerCachePath).ConfigureAwait(false);
-
-            // Read the pointer file again
-            await using var fileStream = System.IO.File.OpenRead(pointerCachePath);
-            return new FileStreamResult(fileStream, MediaTypeNames.Application.Octet);
+            var updatedContentPath = await System.IO.File.ReadAllTextAsync(pointerCachePath).ConfigureAwait(false);
```

Comment:
Why are you reading the file?

Reply:
https://github.com/jellyfin/jellyfin/blob/b49cd1d3017f23fc75703829ac2ea1d45d8a4881/MediaBrowser.Api/ItemLookupService.cs#L285

Looks like this was migrated from the servicestack api